spec/models/tracker_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Tracker, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Would be great to add some simple tests here. Check out some of the other model tests like `spec/models/contributor_spec.rb` for examples of association and validation tests.
You can test that enum with something like this: `it { is_expected.to define_enum_for(:category).with(Identifier.categories.keys) }`